wlcreate/Start-Asian-Love
# Resolves issue 12. As a search engine, I want to see metadata for me to understand and display what Start Asian Love is about

## Description

Related Issue: #27 

In this PR I added metadata for the page title, description, and related Open Graph/Facebook and Twitter information specifically for the Home page. This is because I want to discuss what we think the metadata for each page should be before continuing to add, and to see if there is any feedback. Furthermore, I continued to clean up the file structure, specifically by creating folders to match the components folder and organizing the images in the public folder.

Update:
Since opening this PR I have added a favicon and metadata for every page.

Resources:
* [Next.js Metadata](https://nextjs.org/learn/basics/assets-metadata-css/metadata)
* [Next.js Head](https://nextjs.org/docs/api-reference/next/head)
* https://www.netlify.com/blog/2020/05/08/improve-your-seo-and-social-sharing-cards-with-next.js/
* https://focux.dev/post/generating-open-graph-images-at-build-time-for-your-nextjs-blog
* https://neilpatel.com/blog/open-graph-meta-tags/
* [Official Open Graph Protocol](https://ogp.me)
* [Google advice on title and description](https://developers.google.com/search/docs/advanced/appearance/good-titles-snippets)
* [Images in Public folder](https://stackoverflow.com/questions/60797746/nextjs-images-in-public-folder-not-found-on-deploy-but-are-found-locally)
* [Favicon](https://favicon.io/emoji-favicons/yellow-heart)

## Acceptance Criteria


Update the following meta tags:
- page title
- favicon
- description
- Open Graph (image, title, description, url, and type)

## Self-check

Please check all that apply:



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓  | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

### After

## Testing Steps / QA Criteria


* switch to the `metadata` branch
* `npm run dev`
* when loading each page, check in the developer tools if the metadata is rendered
